infra/gcp/ensure-main-project.sh

```diff
@@ -67,6 +70,9 @@ enable_api "${PROJECT}" storage-component.googleapis.com
 color 6 "Enabling the OSLogin API"
 enable_api "${PROJECT}" oslogin.googleapis.com
 
+color 6 "Ensuring the cluster terraform-state bucket exists"
+ensure_gcs_bucket "${PROJECT}" "${CLUSTER_TERRAFORM_BUCKET}"
```

Comment:
Won't this make a world readable bucket?

Reply:
I didn't think that was a problem, but looking at the contents, maybe it is?

I will change it to private.  Slightly bigger change.